### PR TITLE
Blindness disability in char creation + white cane.

### PR DESCRIPTION
**Pull request by Greenjoe12345**
This PR adds blindness as a select-able disability in the character setup screen. It also adds a white cane item. As well as that it adds the crutch to the loadout. (the white cane is there too). Also allows you to setup your character to spawn without eyes, as a different way for them to be blind. (technically, you can pick both, but they'd just pointlessly stack, I think?)
### IC changelog
Nanotrasen have started to allow blind employees to work aboard the Aurora, doing jobs they are capable of performing.

### CODE REVIEW

**Comment by Greenjoe12345** [comment]
!review

File: code/modules/client/preference_setup/loadout/loadout_general.dm [hunk 1]
```diff
@@ -2,6 +2,14 @@
 	display_name = "cane"
 	path = /obj/item/cane
 
+/datum/gear/cane
+	display_name = "crutch"
+	path = /obj/item/cane/crutch
+
+/datum/gear/cane
```
Comment (mikomyazaki) [inline code/modules/client/preference_setup/loadout/loadout_general.dm:9]: `/datum/gear/cane` is already defined, you need to make these subtypes.

**Review by mikomyazaki — CHANGES_REQUESTED** [review]

File: code/defines/obj/weapon.dm [hunk 2]
```diff
@@ -289,6 +289,12 @@
 	icon_state = "crutch"
 	item_state = "crutch"
 
+/obj/item/cane/white
+	name = "white cane"
+	desc = "A white cane, used by the blind or visually impaired"
```
Comment (Geevies) [inline code/defines/obj/weapon.dm:294]:
```suggestion
	desc = "A white cane, used by the blind or visually impaired."
```

File: code/modules/client/preference_setup/loadout/loadout_general.dm (hunk 1 above)
Comment (Geevies) [reply code/modules/client/preference_setup/loadout/loadout_general.dm:9]: it would be best to make this a selection

Comment (Greenjoe12345) [reply code/modules/client/preference_setup/loadout/loadout_general.dm:9]: oops missed that

File: html/changelogs/greenjoe12345-blindandcane.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscadd: "Adds blindness as a disability to the character setup screen. Adds a white cane. also adds this white cane and the already exsisting crutch item to the loadout."
```
Comment (flpfs) [inline html/changelogs/greenjoe12345-blindandcane.yml:41]:
```suggestion
  - rscadd: "Adds blindness as a disability to the character setup screen. Adds a white cane. Also adds this white cane and the already exsisting crutch item to the loadout."
```

**Comment by Greenjoe12345** [comment]
oop didn't mean config there. meant changelog

**Review by Amunak — APPROVED** [review]

**Review by fernerr — APPROVED** [review]
Technically works. Whether this is desired or not will have to be decided by the maintainers.

**Comment by NonQueueingMatt** [comment]
Will be discussed between maintainers.

**Comment by Arrow768** [comment]
The issue is that we currently do not have a authoritative list that describes what disabilities are permitted in what role.

However until we have such a list that can be used when the question comes up if something is a "reasonable character concept" the PR will not be merged.
Once we have such a list of disabilities, these job restrictions by disability must be mechanically enforced as this can be done with little effort code wise and reduces the burdon on the mods/mins.

Therefore closing this PR for now as @Alberyk will work on that list and a mechanical implementation some time in the foreseeable future.

**Pull request closed without merging**